Sample/SextantSample/Views/HomeView.xaml.cs

```diff
@@ -15,7 +15,7 @@ public HomeView()
 			this.WhenActivated(disposables =>
             {
 				this.BindCommand(ViewModel, x => x.OpenModal, x => x.FirstModalButton).DisposeWith(disposables);
-                this.BindCommand(ViewModel, x => x.PushPage, x => x.PushPage);
+                this.BindCommand(ViewModel, x => x.PushPage, x => x.PushPage).DisposeWith(disposables);
```

Comment:
Is the alignment off here or is just github?

Reply:
Seems it's just github